parcels/particle.py

```diff
@@ -138,6 +138,13 @@ def __setattr__(self, name, value):
         else:
             self._data[name][self._index] = value
 
+    def __getitem__(self, index):
+        self._index = index
+        return self
```

Comment:
This behaviour of `__getitem__` is quite unintuitive, and would be good to rework in a future PR that works on the particledata.

If this was a list, behaviour would be similar to:

```py
>>> lst = [1,2,3,4,5]
>>> lst[:2]
[1, 2]
>>> lst
[1, 2]
>>> lst[:]
[1,2,3,4,5]


```

Reply:
Now also references in https://github.com/OceanParcels/Parcels/issues/2143#issuecomment-3201717596